### ISSUE TITLE
Weston backend for DirectX / DWM

### ISSUE THREAD

**Issue by ohault** [microsoft/wslg#556]
The question of integration between Weston and DWM could open very interesting perspectives.

If additionally RDP shortcut follows its promises for in-memory shortcut, this could be combined to either a bidirectional RDP connection (both client and server) or two opposites connection (Weston -> DWM and Weston <- DWM).

A first point to start could be to start generating DirectX surface from Weston and the other way around.

This could be also promising for running GUI apps inside Containers.

**Comment by hideyukn88** [comment]
@ohault, yes, we do have that plan, see my comments at https://github.com/microsoft/wslg/issues/387#issuecomment-887731973. Thanks!

**Comment by ohault** [comment]
Is there already possible to test it ?
